Python/optimizer_symbols.c

```diff
@@ -121,6 +124,12 @@ _Py_uop_sym_is_null(_Py_UopsSymbol *sym)
     return sym->flags == IS_NULL;
 }
 
+bool
+_Py_uop_sym_is_a_class(_Py_UopsSymbol *sym, unsigned int type_version)
```

Comment:
These need tests.
Specifically tests setting the symbol to be a class, and then setting its class's version.